tools/dhis2-audit-data-extractor/extract_audit.py

```diff
@@ -1,11 +1,21 @@
+# This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 3 of the License.
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>. 
+
+from ast import arg
+from curses.ascii import NUL
+from fileinput import filename
 import psycopg2
 import json
 import gzip
 import configparser
 import os
-from memory_profiler import profile
-import pandas as pd
-from sqlalchemy import create_engine
+#from memory_profiler import profile
+#import pandas as pd
+#from sqlalchemy import create_engine
+import argparse
+from datetime import datetime
+import csv
 
 DHIS2_HOME = os.getenv("DHIS2_HOME", "/home/dhis")
 DHIS2_CONF_FILE =  "{0}/config/dhis.conf".format(DHIS2_HOME)
```

Comment:
As a future improvement, it'd be nice if we check early on if `DHIS2_HOME` and `DHIS2_CONF_FILE` exist and if not output an error to the user.

Reply:
Added in the TODO list, thanks!